src/App.tsx

```diff
@@ -1,18 +1,52 @@
-import React from 'react';
+import React, { useState } from 'react';
 import './App.css';
+import { Pagination } from './components/Pagination';
 import { getNumbers } from './utils';
 
-// eslint-disable-next-line @typescript-eslint/no-unused-vars
-const items = getNumbers(1, 42)
-  .map(n => `Item ${n}`);
-
 export const App: React.FC = () => {
+  const initialCurrentPage = 1;
+  const initialItemsPerPage = 5;
+  const [itemsPerPage, setItemsPerPage] = useState(initialItemsPerPage);
+  const [currentPage, setCurrentPage] = useState(initialCurrentPage);
+  const onPageChange = (page: number) => {
+    setCurrentPage(page);
+  };
+
+  const total = 42;
+  const secondItem = currentPage * itemsPerPage;
+  const firstItem = secondItem - itemsPerPage + 1;
+  const getCorrectedFirstItem = firstItem === 0 ? firstItem + 1 : firstItem;
+  const getCorrectSecondItem = secondItem > total ? total : secondItem;
```

Comment:
Consider renaming 'getCorrectSecondItem' to 'correctedSecondItem' for consistency with the naming convention.